rubix-tests/src/test/robotframework/acceptance/bks.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Library     Collections
+Library     OperatingSystem
+Library     Process
+Library     com.qubole.rubix.client.robotframework.BookKeeperClientRFLibrary
+
+*** Keywords ***
+Start BKS
+    [Arguments]         &{options}
+    ${optionArgs} =     Get options argument    &{options}
+    Run                 ${CURDIR}${/}bks.sh start ${optionArgs}
+    Sleep               1s
+
+Stop BKS
+    [Arguments]         &{options}
+    ${optionArgs} =     Get options argument   &{options}
+    Run                 ${CURDIR}${/}bks.sh stop ${optionArgs}
+
+Get options argument
+    [Arguments]         &{options}
+    @{optionsList} =    Create List
+    :FOR    ${key}      IN      @{options.keys()}
+    \   ${value} =      Get From Dictionary     ${options}    ${key}
+    \   Append To List  ${optionsList}  -D${key}\=${value}
+    ${optionArgs} =     Join Command Line   ${optionsList}
+    [Return]            ${optionArgs}
+
+Generate test files
+    [Arguments]     ${numFiles}
+    ...             ${fileLength}
+    @{testFileList} =   Create List
+    :FOR    ${index}    IN RANGE    ${numFiles}
+    \   ${testFile} =        Set variable  ${REMOTE_PATH}${index}
```

Comment:
Where are we deleting the test file after the test is finished ?

Reply:
Files are now deleted as part of test teardown (line 75 below)
https://github.com/qubole/rubix/blob/ae9f82a296250d5ff92c399afab1c3e7b4b14d35/rubix-tests/src/test/robotframework/acceptance/CacheEviction.robot#L73-L75